tuplex/codegen/include/BlockGeneratorVisitor.h

```diff
@@ -344,6 +345,8 @@ namespace codegen {
         // store current iteration ending block and loop ending block for for and while loops
         std::deque<llvm::BasicBlock*> _loopBlockStack;
 
+        std::shared_ptr<IteratorContextProxy> _icp;
```

Comment:
let's try to avoid acronyms here, just call the variable _iteratorContentProxy. Then it's super clear what it is :)

Reply:
Refactored variable name.